lib/api/objectGetLegalHold.js

```diff
@@ -61,10 +63,14 @@ function objectGetLegalHold(authInfo, request, log, callback) {
                     if (versionId) {
                         log.trace('requested version is delete marker',
                             { method: 'objectGetLegalHold' });
+                        // FIXME wee should return a `x-amz-delete-marker: true` header,
```

Comment:
should this be TODOs so they are more easily searchable? also wee -> we on a few of these

Reply:
since it's an actual bug, I prefer FIXME over TODO (why TODO is more easily searchable btw?)

Yeah I will fix the typo